EasyFinance.Application/Features/CategoryService/CategoryService.cs

```diff
@@ -66,6 +67,26 @@ public async Task<ICollection<Category>> GetAsync(Guid projectId, DateTime from,
             .Where(c => !c.Archive)
             .ToList();
 
+        public async Task<ICollection<DefaultCategory>> GetDefaultCategoriesAsync(Guid projectId)
+        {
+            if (projectId == Guid.Empty)
+                throw new ArgumentNullException(nameof(projectId));
+
+            var project = await unitOfWork.ProjectRepository
+                .Trackable()
```

Comment:
You never use this entities after to update some data. Because of that you don't need to track modifications.
```suggestion
                .NoTrackable()
```